meta/main.yml

```diff
@@ -14,10 +14,15 @@ galaxy_info:
         - 7
     - name: Ubuntu
       versions:
-        - all
+        - trusty
+        - xenial
+        - bionic
+        - cosmic
```

Comment:
My roles don't support non-LTS releases of Ubuntu. Too many annoying edge cases to support :)

So this list should currently be trusty, xenial, bionic.